.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -268,8 +268,8 @@ export const CoursePlannerEdit = () => {
   const container1EmptyContent = () => {
     return (
       <>
-        <Text as="div" className="!mb-8">
-          <p>There are no more available projects to create your course plan</p>
+        <Text className="mb-8 max-w-xl">
+          There are no more available projects to create your course plan
```

Comment:
As part of the Cookbook RFC work, we will review such stories and make the text and such more generic (and use that FPO block)

Reply:
Super down for that